<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fedora Gooey Karma</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QWidget" name="pkgLists" native="true">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="searchEdit">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Search packages...</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="installedBtn">
             <property name="text">
              <string>Installed</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="availableBtn">
             <property name="text">
              <string>Available</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="pkgList">
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="pkgNameLabel">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Package name</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="font">
             <font>
              <family>Monospace</family>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTreeWidget" name="treeWidget_bugs">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="columnCount">
               <number>2</number>
              </property>
              <column>
               <property name="text">
                <string>Bug id</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Title</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeWidget_test_cases">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <column>
               <property name="text">
                <string>Test cases</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeWidget_related_packages">
              <column>
               <property name="text">
                <string>Related packages</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget_feedback">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="headerMinimumSectionSize">
           <number>30</number>
          </attribute>
          <column>
           <property name="text">
            <string>Karma</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>User</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Comment</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="karmaWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="usernameEdit">
             <property name="placeholderText">
              <string>username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordEdit">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="okBtn">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancelBtn">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="commentEdit"/>
           </item>
           <item>
            <widget class="QComboBox" name="karmaBox">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <item>
              <property name="text">
               <string>-1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+1</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="sendBtn">
             <property name="text">
              <string>Send Karma</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder></zorder>
   <zorder>karmaWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>996</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
